src/helpers.js

```diff
@@ -74,14 +74,15 @@ export function getDependencies(packageName: string): Array<Dependency> {
   return toReturn
 }
 
-export function promptUser(packageName: string, dependencies: Array<Dependency>): Promise<'Yes' | 'No' | 'Never'> {
+export async function promptUser(packageName: string, dependencies: Array<Dependency>): Promise<'Yes' | 'No' | 'Never'> {
   const configPath = Path.join(atom.getConfigDirPath(), 'package-deps-state.json')
-  const configFile = new ConfigFile(configPath, { ignored: [] })
-  if (configFile.get('ignored').includes(packageName)) {
+  const configFile = await ConfigFile.get(configPath, { ignored: [] }, { createIfNonExistent: true })
+  const ignoredPackages = await configFile.get('ignored')
+  if (ignoredPackages.includes(packageName)) {
     return Promise.resolve('No')
   }
 
-  return new Promise(function(resolve) {
+  return await new Promise(function(resolve) {
```

Comment:
Since this is an `async` function, putting `await` here is redundant as the calling function will already be expecting a Promise that it needs to wait on.